akka-actor-typed-tests/src/test/scala/akka/actor/typed/scaladsl/ActorLoggingSpec.scala

```diff
@@ -250,6 +252,26 @@ class ActorLoggingSpec extends ScalaTestWithActorTestKit("""
       }
     }
 
+    "pass tags from props to MDC" in {
+      val behavior = Behaviors.setup[String] { ctx =>
+        ctx.log.info("Starting up")
+
+        Behaviors.receiveMessage {
+          case msg =>
+            ctx.log.info("Got message {}", msg)
```

Comment:
Just a nit, up to you
```suggestion
            ctx.log.info("Received message {}", msg)
```